mysql-client-tests/mysql-client-tests.bats

```diff
@@ -48,3 +48,8 @@ teardown() {
     (cd $BATS_TEST_DIRNAME/c; make clean; make)
     $BATS_TEST_DIRNAME/c/mysql-connector-c-test $USER $PORT $REPO_NAME
 }
+
+@test "dotnet mysql connector" {
+    cd $BATS_TEST_DIRNAME/dotnet/
+    dotnet run -- $USER $PORT $REPO_NAME 3>&-
```

Comment:
I know this is mine but I would add a comment:

`# dotnet run uses output channel 3 which conflicts with bats so we pipe it to null`